### PR TITLE
Deprecate WallID as direct child under Side

### LINKED ISSUE #233: Deprecate auc:WallID as direct child of auc:Side
Now that `auc:WallIDs` was added in #223 , remove the choice to use`auc:WallID` as a direct child of `auc:Side` in schema v3.0.

### PR DESCRIPTION
**Pull request by JieXiong9119**
See proposal.

Resolves #233

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -1179,7 +1179,6 @@
                                 </xs:complexType>
                               </xs:element>
                               <xs:choice>
```
[the remark this answers is not in the document]
Comment (macintoshpie) [reply BuildingSync.xsd:1181]: We should remove `xs:choice`, Mark's comment was saying he preferred that users could choose between WallID or WallIDs. But we are removing that choice completely now, so the `xs:choice` element is no longer needed.

**Review by macintoshpie — CHANGES_REQUESTED** [review]

File: BuildingSync.xsd (hunk 1 above)
[the remark this answers is not in the document]
Comment (JieXiong9119) [reply BuildingSync.xsd]: Make sense. I removed `xs:choice`.
Another issue is that deleting the choice of directly using WallID causes validation failures to all examples, as they all don't use `WallIDs`.

File: proposals/2021/Deprecate WallID.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+On the other hand, to avoid breaking change for Audit Template, the `xsd:choice` element will remain.
+
+## Implementation
+
+```xml
+                        <xs:element name="Side" maxOccurs="unbounded">
+                          <xs:complexType>
+                            <xs:sequence>
+                              ...
+                              <xs:choice>
```
Comment (macintoshpie) [inline proposals/2021/Deprecate WallID.md:19]: Thanks, looks like we need to remove choice here too

Comment (JieXiong9119) [reply proposals/2021/Deprecate WallID.md]: You're right. I just update the proposal.

File: proposals/2021/Deprecate WallID.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ This proposal is to deprecate the choice of single `auc:WallID` reference as dir
 
 ## Justification
 
-`WallIDs` was added as an `xsd:choice` element in [#223](https://github.com/BuildingSync/schema/pull/223) to allow multiple `auc:WallID`s referenced for a `auc:Side` element (unbounded). This made the single `auc:WallID` reference choice redundant, so we propose to remove it.
-On the other hand, to avoid breaking change for Audit Template, the `xsd:choice` element will remain.
+`auc:WallIDs` was added as an `xsd:choice` element in [#223](https://github.com/BuildingSync/schema/pull/223) to allow multiple `auc:WallID`s referenced for a `auc:Side` element (unbounded). This made the single `auc:WallID` reference choice redundant, so we propose to remove it.
+In addition, as `auc:WallIDs` becomes the only choice, the `xsd:choice` element is no longer needed, so we propose to remove this layer and make `auc:WallIDs` direct child under `auc:Side`.
```
Comment (macintoshpie) [inline proposals/2021/Deprecate WallID.md:10]: nice 👍

**Review by macintoshpie — APPROVED** [review]

**Comment by nllong** [comment]
@JieXiong9119 Are you planning on updating the examples so these tests will pass? Or are we waiting for @macintoshpie 's translator?

**Comment by JieXiong9119** [comment]
> @JieXiong9119 Are you planning on updating the examples so these tests will pass? Or are we waiting for @macintoshpie 's translator?

I'll probably update the examples first.

File: BuildingSync.xsd [hunk 4]
```diff
@@ -1123,16 +1123,13 @@
...
-                                <xs:element ref="auc:WallID" minOccurs="0"/>
-                                <xs:element name="WallIDs">
-                                  <xs:complexType>
-                                    <xs:sequence>
-                                      <xs:element ref="auc:WallID" minOccurs="0" maxOccurs="unbounded"/>
-                                    </xs:sequence>
-                                  </xs:complexType>
-                                </xs:element>
-                              </xs:choice>
+                              <xs:element name="WallIDs">
```
Comment (nllong) [inline BuildingSync.xsd:1126]:
```suggestion
                              <xs:element name="WallIDs" minOccurs="0" maxOccurs="1">
```

**Comment by macintoshpie** [comment]
@JieXiong9119 can you resolve these conflicts? Once that's done we should be good to merge

**Comment by JieXiong9119** [comment]
> @JieXiong9119 can you resolve these conflicts? Once that's done we should be good to merge

Just did. Could you check one more time and merge it if it looks good?

**Review by macintoshpie — APPROVED** [review]
Looks good 👍

**Pull request merged**